src/main/webapp/index.html

```diff
@@ -3,9 +3,13 @@
     <meta charset="UTF-8">
     <title>Home</title>
     <link rel="stylesheet" href="style.css">
+    <script src="startup.js"></script>
```

Comment:
Suggestion: a good idea is to find a good name for that, as the other steps will also have separate scripts. But I guess we can think about this later, as soon as we begin working on the other pages!

Reply:
Good point. I think since this is the first page we can consider this to be the start up and the try to find suggestive names for the next steps